components/assetlist/stories/template.js

```diff
@@ -16,6 +16,7 @@ export const AssetListItem = ({
   image,
   iconName,
   label,
+  isNavigated = false,
```

Comment:
Does this need to be added to the assetlist.stories.js file as well?

Reply:
I thought about that. I don't know! Current site does not have it and it might warrant a separate story in AssetList for it to make sense and display properly. Let me see if I can create a helpful story for it.

Reply:
I'm just going to add an additional list item that `isNavigated` 😄